src/newViews/ProjectEdit/GeoAreas/CustomGeoAddForm/index.tsx

```diff
@@ -25,6 +25,8 @@ type FormType = {
     title: string;
     project: number;
     code?: string;
+    public: boolean;
+    isPublished: boolean;
```

Comment:
```suggestion
    public?: boolean;
    isPublished?: boolean;
```

We should check this with backend